### PR TITLE
Refactor of RM worker and add features to improve RM

### PR DESCRIPTION
**Pull request by Branislav-Karkus**
Refactor of resource-manager-worker, add features to util.py so there will be completed and failed response with logs, add logging handler because its one of the dependencies for new RM worker and changed version in setup.py to 1.1.2

### CODE REVIEW

File: frinx_conductor_workers/frinx_conductor_workers/resource_manager_worker.py [hunk 1]
```diff
@@ -146,15 +148,15 @@ def claim_resource(task, logs):
     """
```
Comment (Vasyl-9) [inline frinx_conductor_workers/frinx_conductor_workers/resource_manager_worker.py:148]: add AlternativeId to response_body in docs

File: frinx_conductor_workers/frinx_conductor_workers/resource_manager_worker.py [hunk 2]
```diff
@@ -213,7 +215,7 @@ def query_claimed_resources(task, logs):
         None if "alternativeId" not in task["inputData"] else task["inputData"]["alternativeId"]
     )
     if pool_id is None:
-        return failed_response(logs, "No pool id")
+        return failed_response_with_logs(logs, "No pool id")
     variables = {"pool_id": pool_id}
     if alternative_id is not None:
```
Comment (Vasyl-9) [inline frinx_conductor_workers/frinx_conductor_workers/resource_manager_worker.py:220]: if alternative_id is not None and len(alternative_id) > 0:

File: frinx_conductor_workers/frinx_conductor_workers/uniconfig_worker.py [hunk 3]
```diff
@@ -389,7 +389,6 @@ def create_tx_multizone(task):
     for d in devices_by_uniconfig:
         response = create_tx_internal(uniconfig_cluster=d.uc_cluster)
         if response["status"] != util.COMPLETED_STATUS:
-
```
Comment (Vasyl-9) [inline frinx_conductor_workers/frinx_conductor_workers/uniconfig_worker.py:392]: Why did you change this file?

File: frinx_python_sdk/src/frinx/workers/uniconfig/uniconfig_worker.py [hunk 4]
```diff
@@ -389,7 +389,6 @@ def create_tx_multizone(task):
     for d in devices_by_uniconfig:
         response = create_tx_internal(uniconfig_cluster=d.uc_cluster)
         if response["status"] != task_response.COMPLETED_STATUS:
-
```
Comment (Vasyl-9) [inline frinx_python_sdk/src/frinx/workers/uniconfig/uniconfig_worker.py:392]: Why did you change this file?

**Review by Vasyl-9 — CHANGES_REQUESTED** [review]

File: frinx_conductor_workers/frinx_conductor_workers/uniconfig_worker.py (hunk 3 above)
Comment (Branislav-Karkus) [reply frinx_conductor_workers/frinx_conductor_workers/uniconfig_worker.py:392]: new version of black, which formatted even this file

File: frinx_python_sdk/src/frinx/workers/uniconfig/uniconfig_worker.py (hunk 4 above)
Comment (Branislav-Karkus) [reply frinx_python_sdk/src/frinx/workers/uniconfig/uniconfig_worker.py:392]: new version of black, which formatted even this file

File: frinx_conductor_workers/frinx_conductor_workers/uniconfig_worker.py (hunk 3 above)
Comment (Vasyl-9) [reply frinx_conductor_workers/frinx_conductor_workers/uniconfig_worker.py:392]: ok

File: frinx_python_sdk/src/frinx/workers/uniconfig/uniconfig_worker.py (hunk 4 above)
Comment (Vasyl-9) [reply frinx_python_sdk/src/frinx/workers/uniconfig/uniconfig_worker.py:392]: ok

File: frinx_conductor_workers/frinx_conductor_workers/resource_manager_worker.py (hunk 1 above)
Comment (Branislav-Karkus) [reply frinx_conductor_workers/frinx_conductor_workers/resource_manager_worker.py:152]: done

File: frinx_conductor_workers/frinx_conductor_workers/resource_manager_worker.py (hunk 2 above)
Comment (Branislav-Karkus) [reply frinx_conductor_workers/frinx_conductor_workers/resource_manager_worker.py]: done

**Review by Vasyl-9 — APPROVED** [review]

**Review by Vasyl-9 — APPROVED** [review]

**Review by Jozefiel — APPROVED** [review]

**Pull request merged**